contracts/dao-core/src/msg.rs

```diff
@@ -231,6 +231,64 @@ pub enum QueryMsg {
 
 #[cw_serde]
 pub enum MigrateMsg {
-    FromV1 { dao_uri: Option<String> },
+    FromV1 {
+        dao_uri: Option<String>,
+        params: Option<Box<MigrateParams>>,
+    },
     FromCompatible {},
 }
+
+#[cw_serde]
```

Comment:
could we perhaps move these types into a new file `migrate_msg.rs` and add a comment at the top of the file explaining where these are taken from?

```
//! types for migrating the core module to version two. taken from ...
```